File: generate.core/Models/App/ToggleQuestion.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace generate.core.Models.App
{
    public class ToggleQuestion
    {
        public int ToggleQuestionId { get; set; }
        public int ToggleQuestionTypeId { get; set; }
        public int ToggleSectionId { get; set; }
        public string EmapsQuestionAbbrv { get; set; }
        public string QuestionText { get; set; }
        public int QuestionSequence { get; set; }
        public int? ParentToggleQuestionId { get; set; }
        public ToggleQuestionType ToggleQuestionType { get; set; }
        public ToggleSection ToggleSection { get; set; }

        [ForeignKey("ParentToggleQuestionId")]
        public virtual ToggleQuestion ParentToggleQuestion { get; set; }
    }
}

File: generate.web/ClientApp/src/app/models/app/toggleQuestion.ts
import {ToggleQuestionType} from './toggleQuestionType';

export interface ToggleQuestion {
    toggleQuestionId: number;
    toggleQuestionTypeId: number;
    toggleSectionId: number;
    emapsQuestionAbbrv: string;
    questionText: string;
    parentToggleQuestionId: number;
    toggleQuestionType: ToggleQuestionType;
}